### ISSUE TITLE
syntax error when importing

### ISSUE THREAD

**Issue by jsadler2** [aerispaha/swmmio#17]
Hi,
when I do `from swmmio import swmmio` I get a syntax error coming from the `definitions` package:
```python
In [1]: from swmmio import swmmio
  File "C:\Users\jeff\Anaconda2\lib\site-packages\definitions\parser.py", line 83
    yield from cls._flat_tree(element)
             ^
SyntaxError: invalid syntax
```
I am running python 2.7. From what I can tell, the `yield from` syntax is only valid in python 3. Have you had this problem or know of a work around?

**Comment by aerispaha** [comment]
Hey @jsadler2, I haven't had this issue. What is the 'definitions' package you have installed in your root site-packages? There's a chance that that package conflicts with the definitions.py module within the root directory of swmmio. 

Generally, I want to clean up the package structure swmmio, and I'm fairly certain definitions.py will be on the chopping block because I don't think it follows (any) good conventions.

**Comment by jsadler2** [comment]
Thanks for getting back to me @aerispaha. I think I found the problem. When I install `swmmio` from pypi, the `definitions.py` file is not included in the package somehow so it throws the error ```No module named definitions```. Without looking into it much, I thought that `definitions` was a third-party library on which swmmio was dependent so I ran `pip install definitions` and the `definitions` package on pypi is the one throwing the syntax error in the OP.

When I clone the repo, instead of going through pypi, and do `from swmmio import swmmio` it works without error.
